### PR TITLE
SURF-553 Edited content for alerts

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by asettle** [comment]
@robb-romans - I was basing most of my changes off what you did too, could you double check for consistency? :)

File: _component_design/alert-bar.md [hunk 1]
```diff
@@ -14,77 +14,110 @@ last-modified: 2017-08-17
...
-<div class="hxRow">
-{% column left:"hxCol-4" %}
-The alert bar consists of 3 major parts ALWAYS:
+* Matches the margin of the content area to which it relates.
 
-- Bar: enclosing shape to frame and present the alert
-- Icon: which provides context to the classification of the alert type
-- Message: provides details and/or instructions to give the user context
+{% figure [caption:"When you need to describe an image in greater detail, use this
+ block of text so that you can keep the left area clutter-free. 3 lines or less."] [class:"image bg-light border"] %}
```
Comment (araiford) [inline _component_design/alert-bar.md]: nix the placeholder caption.  Sorry bout these -- we'll make sure the designers remove them when they don't need them.

File: _component_design/alert-bar.md [hunk 2]
```diff
@@ -14,77 +14,110 @@ last-modified: 2017-08-17
...
 
-## Inline
+The alert bar pattern seeks to provide a user with at a glance status of a
+given content section. The basic pattern can display four types of status:
 
-Provides info for action in context to the OBJECT they are ACTIVELY doing. User
-understands this is in relation to the context they're actively doing and
-provides the most specificity.
+{% figure [caption:"When you need to describe an image in greater detail, use this
+ block of text so that you can keep the left area clutter-free. 3 lines or less."] [class:"image bg-light border"] %}
```
Comment (araiford) [inline _component_design/alert-bar.md]: nix the placeholder caption.

Comment (araiford) [inline _component_design/alert-bar.md]: nix the placeholder caption.

File: _component_design/alert-bar.md [hunk 3]
```diff
@@ -14,77 +14,110 @@ last-modified: 2017-08-17
...
-## Alert Bar
+## Variations
 
-Provides info for action in context to the PAGE/REGION. User understands this
-is in relation to the context they're currently working on holistically.
+Each status corrolates to a specific color in the Helix color palette.
 
-## Toast
+{% figure [caption:"Alert bar status types"] [class:"image bg-light border"] %}
+ ![]({{site.url}}/assets/images/components/content-areas/alerts/alert-bar-status-types.svg){:width="80%"}
```
Comment (araiford) [inline _component_design/alert-bar.md]: this image should be alert-bar-variations.svg

**Pull request merged**